From 3466c54a1adfe75dd196c483f082f37b9768f20a Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Fri, 19 May 2017 17:15:23 -0400
Subject: [PATCH 1694/2094] drm/amd/display: No need to assert on stream_status

This will be NULL on a new stream. DC handles it gracefully.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
index f0d17aa..938372b 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
@@ -3065,9 +3065,6 @@ static enum surface_update_type  amdgpu_dm_check_surfaces_update_type(
 			dc_stream_get_status(dc_stream);
 	enum surface_update_type update_type;
 
-	ASSERT(stream_status);
-
-
 	memset(srf_updates, 0, sizeof(srf_updates));
 	memset(flip_addr, 0, sizeof(flip_addr));
 	memset(plane_info, 0, sizeof(plane_info));
-- 
2.7.4

